rest/api_test.go

```diff
@@ -2275,6 +2276,7 @@ func TestHandleStats(t *testing.T) {
 
 // TestHideProductInfo ensures that detailed product info is not shown on non-admin REST API responses if set.
 func TestHideProductInfo(t *testing.T) {
+	t.Skip("here")
```

Comment:
Unclear what's changed that's requiring this to be skipped.

Reply:
mistake